content/en/blog/2021/external-locality-failover/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Configuring failover for external services"
+description: Learn how to configure locality load balancing and failover for endpoints that are outside of your mesh.
+publishdate: 2021-06-04
+attribution: "Ram Vennam (Solo.io)"
+keywords: [locality,region,failover,Istio,outlier,external]
+---
+
+Istio’s powerful APIs can be used to solve a variety of service mesh use cases. Many users know about it’s strong ingress and east-west capabilities but it also offers many features for egress (outgoing) traffic. This is especially useful when your application needs to talk to an external service - such as a database endpoint provided by a cloud provider. There are often choices of [multiple endpoints](https://docs.aws.amazon.com/general/latest/gr/ddb.html) depending on where your workload is running. You typically want to choose the endpoint closest to your workload for latency reasons, but you may need to configure automatic failover to another endpoint in case things are not working as expected.
```

Comment:
> about it’s

its

> There are often choices of [multiple endpoints](https://docs.aws.amazon.com/general/latest/gr/ddb.html)

Only on clouds without good anycast load balancers :)
That's not a general link for "multiple endpoints", so it might pay to introduce the fact you're talking about DynamoDB earlier.

"For example, Amazon's DynamoDB offers different endpoints depending on where your workload is running".

(I assume it's different endpoints to the same global service, not regional slices of it?)

> in case

in _the_ case?  (unsure)